app/config.py

```diff
@@ -7,18 +7,14 @@ class Config():
     NOTIFICATION_QUEUE_PREFIX = os.getenv('NOTIFICATION_QUEUE_PREFIX')
     QUEUE_NAME = 'govuk-alerts'
 
-    # Logging
-    DEBUG = False
-    LOGGING_STDOUT_JSON = os.getenv('LOGGING_STDOUT_JSON') == '1'
-
     NOTIFY_APP_NAME = 'govuk-alerts'
     NOTIFY_AWS_REGION = 'eu-west-1'
     NOTIFY_LOG_PATH = os.getenv('NOTIFY_LOG_PATH', 'application.log')
 
     BROADCASTS_AWS_ACCESS_KEY_ID = os.getenv("BROADCASTS_AWS_ACCESS_KEY_ID")
     BROADCASTS_AWS_SECRET_ACCESS_KEY = os.getenv("BROADCASTS_AWS_SECRET_ACCESS_KEY")
     BROADCASTS_AWS_REGION = 'eu-west-2'
-    GOVUK_ALERTS_BUCKET_NAME = os.getenv("GOVUK_ALERTS_BUCKET_NAME")
+    GOVUK_ALERTS_S3_BUCKET_NAME = os.getenv("GOVUK_ALERTS_S3_BUCKET_NAME")
```

Comment:
Has the new env var been updated in the manifest/secrets so that it gets picked up correctly?

Reply:
Yeah, the "_S3_" one matches [the one in the manifest](https://github.com/alphagov/notifications-govuk-alerts/blob/main/manifest.yml.j2#L46), which already matched the one in the credentials repo. I thought it would be easier to take the renaming in this direction than the other way.